internal/services/containers/kubernetes_cluster_network_resource_test.go

```diff
@@ -58,6 +58,21 @@ func TestAccKubernetesCluster_serviceMeshProfile(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesCluster_serviceMeshProfileWithCertificateAuthority(t *testing.T) {
```

Comment:
Since this is touching add-on functionality would you mind moving this test into the `kubernetes_cluster_addons_resource_test.go` and follow the test naming convention in that file?

Reply:
Updated in latest commit